lib/internal/modules/cjs/loader.js

```diff
@@ -85,6 +85,20 @@ const relativeResolveCache = Object.create(null);
 
 let requireDepth = 0;
 let statCache = null;
+
+function enrichCJSError(err) {
+  const stack = err.stack.split('\n');
+
+  const lineWithErr = stack[1];
+
+  if (err.message.startsWith('Unexpected token export') ||
+    (/^\bimport\b(?=[\s\t\\(\\{\\'\\"\\*\n])?(as)?(from)?/gi).test(lineWithErr)) {
```

Comment:
1. We don't want the regular expression to be case insensitive because `IMPORT` is not valid import syntax. So we should remove the `i`.
2. `\b` can be replaced with `\s*` I think mostly here
3. I think this is still matching the string - `"import"`, which is wrong, because we specifically don't want to match `import(invalid')`. Perhaps include a test for this.